### DISCUSSION TITLE
react-hook-form.com is reloading from time to time

### DISCUSSION THREAD

**Discussion by pooyarm** [react-hook-form/react-hook-form#7437] [General] [answered]
Is it only for me or it happens for others? react-hook-form.com gets reloaded again and again from time to time, and after a while it's in an infinite reload loop!

I use chrome, was using adblocker, but disabled it, still having the issue.

**Comment by bluebill1049** [comment]
Sounds like a useEffect issue

**Reply by pooyarm** [reply]
> Sounds like a useEffect issue

Thanks for quick reply, as I debugged it looks like it was apollo client extension who was doing the reload. idk why only on react-hook-form, and idk why it happens even when devtools is closed!.

anyway I've disabled it, so far looks ok, I'll close this discussion, if it didn't happen in few next hours.

**Comment by pooyarm** [comment, marked as answer]
The problem was a conflict with apollo client extension, and it was triggered on search functionality of the website, it works fine with disabling that extension.
